milli/Cargo.toml

```diff
@@ -33,7 +34,7 @@ serde = { version = "1.0.123", features = ["derive"] }
 serde_json = { version = "1.0.62", features = ["preserve_order"] }
 slice-group-by = "0.2.6"
 smallstr =  { version = "0.2.0", features = ["serde"] }
-smallvec = "1.6.1"
+smallvec = { version = "1.6.1", features = ["write"] }
```

Comment:
Do we need this? I think we can remove the `write` feature now.

Reply:
done